sxt/field51/type/element.h

```diff
@@ -32,19 +32,16 @@ class element {
 
```

Comment:
We should also be able to drop the coda_callable.h inclusion and dependency.

Reply:
Good idea. I was also able to remove the header from the other element components in the `field12` and `field25` packages. If that goes too far, I can revert that update.